tests/haskell/GhcModBearTest.py

```diff
@@ -0,0 +1,51 @@
+from bears.haskell.GhcModBear import GhcModBear
+from coalib.testing.LocalBearTestHelper import verify_local_bear
+
+
+good_file1 = """
```

Comment:
every testfile should have a purpose, give them proper names so people see the purpose in the name and they don't have to thing so much ;)